src/compiler/types.rs

```diff
@@ -1059,6 +1086,24 @@ pub struct EmptyStatement {
     pub _node: BaseNode,
 }
 
+#[derive(Debug)]
+#[ast_type(ancestors = "Statement")]
+pub struct Block {
+    _node: BaseNode,
+    pub statements: NodeArray, /*<Statement>*/
+    pub(crate) multi_line: Option<bool>,
```

Comment:
Some Typescript fields are marked `@internal` which I'm not sure exactly what it means (guessing it suppresses generating documentation for things?) but so `pub(crate)` seemed like possibly the appropriate corresponding "visibility"